docs/priority/README.md

```diff
@@ -1,18 +1,26 @@
 # Workload Priority
 In Kubernetes, assigning different priorities to workloads ensures efficient resource management, minimizes service disruption, and supports better scaling. 
-It allows critical applications to receive the resources they need before less important ones, ensuring high availability and compliance with SLAs. 
-By prioritizing workloads, KAI Scheduler can preempt lower-priority workloads to make room for higher-priority ones. 
+It allows critical applications to receive the resources they need before less important ones, ensuring high availability and compliance with SLAs.
+By prioritizing workloads, KAI Scheduler schedules jobs according to their assigned priority. 
+When sufficient resources aren't available for a workload, the scheduler can preempt lower-priority workloads to free up resources for higher-priority ones.
 This approach ensures that mission-critical services are always prioritized in resource allocation.
```

Comment:
I think we need to explain that priority affects the priority inside the queue, and between queues, resources are divided according to DRF - meaning that a workload with a lower priority can potentially preempt a higher priority one, if reclaim permits

Reply:
Partially exists in this doc:
https://github.com/NVIDIA/KAI-Scheduler/tree/main/docs/fairness

Reply:
The topic of this one is workload priority